docs/src/installation/deployment.md

```diff
@@ -574,11 +574,68 @@ If you decided to set up a dex server, you can parametrize the [metal role](http
 
 ## Bootstrapping a Partition
 
-Documentation will follow soon. Please come to our Slack channel when you have come to this point and require any help.
+### Management network
 
-## Partition Deployment
+To be able to deploy and maintain a metal-stack partition, you need to bootstrap the management network first. 
+Some considerations must be made to fullfill the requirements of our infrastructure, a partition is designed to be:
+
+- secure
+- fully routable (BGP)
+- scalable
+- resilient
+- deployable via CI/CD jobs
+- accessible from the internet from specific IPs
+
+In order to accomplish this task remotely and in a nearly automatic manner, you have to bootstrap the components in this order:
+
+1. Management Firewalls
+2. Management Servers
+3. Management Spines
+4. Management Leaves
+5. Leaves, Spines and Exits
+
+This document assumes that all cabling is done. Here is a quick overview of the architecture:
+
+![Management Net](mgmt_net_layer3.svg)
+
+### Management Firewalls
+
+As you can see, the Management Firewalls are the first bastion hosts in a partition to provide access to our infrastructure. There are two of them in each partition to guarantee high availability and load balancing. The very first configuration of these routers has to be done manually to solve the chicken and egg problem that you need the management firewalls in place to deploy the partition. Manually means that we generate a configuration template with ansible that we deploy with copy/paste, and the load, through the machine console. Once the management server has been deployed, we are able to deploy this configuration via gitlab-runner and ansible. For this you need the user and the ssh-key, which is deployed with the configuration file mentioned above.
```

Comment:
```suggestion
As you can see, the management firewalls are the first bastion hosts in a partition to provide access to your infrastructure. There are two of them in each partition to guarantee high availability and load balancing. 

The very first configuration of these routers are applied manually. As we strive for the highest degree of automation in the data center, this first manual step only provides the minimum requirements to allow the rest of the partition getting deployed through CI. Manually means that we generate a configuration template with Ansible and deploy it through the machine console. Once the management server has been deployed, we are able to deploy the rest of the configuration through CI and Ansible (e.g. by deploying a Gitlab- or Github-Runner on the management server). 

For every partition you need to generate an SSH key, which is deployed with the configuration file mentioned above.
```

The fragment `, and the load` I do not understand. What load? Edited quite a lot on this paragraph but please feel free use whatever you want from this suggestion.

Where is the configuration file "mentioned above"?

Reply:
Mainly the network traffic load but also the load for deployments and other microservices on the management servers.

Reply:
I don't want to publish the firewall configuration. It is made to fit our environment needs.